src/components/Carousel/styled.ts

```diff
@@ -15,13 +15,26 @@ export const ButtonContainer = styled('span', {
     height: BUTTON_SIZE,
     width: BUTTON_SIZE,
     backgroundColor: '$white',
-    boxShadow: '0 5px 8px -8px $shadow12, 0 2px 4px -3px $shadow12, 0 0 0 1px $shadow3, 0 1px 3px 1px $shadow1',
+    color: '$black',
+    boxShadow: '0 1px 3px 1px $shadow1, 0 0 0 1px $shadow3, 0 2px 4px -3px $shadow12, 0 5px 8px -8px $shadow12',
 
     [`& ${Icon.Frame}`]: {
       height: '$xxs',
       width: '$xxs',
       color: 'rgba(0,0,0,0.6)',
     },
+
+    [`&:hover`]: {
+      boxShadow: '0 1px 4px 1px $shadow4, 0 0 0 1px $shadow4, 0 2px 4px -3px $shadow12, 0 5px 8px -8px $shadow12',
+    },
+
+    [`&:active`]: {
+      boxShadow: ' 0 1px 4px 0 $shadow8, 0 0 4px 0 $shadow4, 0 2px 4px 0 $shadow12, 0 5px 8px 0 $shadow12',
+    },
+
+    [`&:hover ${Icon.Frame}, &:active ${Icon.Frame}`]: {
```

Comment:
minor thing, but I've been styling them like this so each rule is easy to read
```suggestion
    [`
      &:hover ${Icon.Frame},
      &:active ${Icon.Frame}
    `]: {
```